common-lib/src/main/scala/com/gu/mediaservice/lib/imaging/ImageOperations.scala

```diff
@@ -160,6 +168,7 @@ class ImageOperations(playPath: String) extends GridLogging {
       // Tidy up any other files (blah-1,2,3 etc will be created for each subsequent layer)
       cleanUpLayerFiles(mainPart, extension, 1)
     }
+    fileBits.last
```

Comment:
should this be the extension of the container or the extracted layer?

Reply:
It will always be the extension of the extracted layer.